src/main/java/com/esri/samples/geometry/convex_hull_list/ConvexHullListSample.java

```diff
@@ -0,0 +1,255 @@
+      graphicsOverlay.getGraphics().add(polygonGraphic2);
+
+      // create a button to create and show the convex hull
+      Button convexHullButton = new Button("Convex Hull");
+      convexHullButton.setMaxWidth(Double.MAX_VALUE);
+
+      // create a button to clear the convex hull results
+      Button resetButton = new Button("Reset");
+      resetButton.setMaxWidth(Double.MAX_VALUE);
+
+      // create a check box for toggling union option on or off
+      CheckBox checkBox = new CheckBox("Union");
+
+      convexHullButton.setOnAction(e -> {
+        // reset the convex hull graphics overlay
+        convexHullGraphicsOverlay.getGraphics().clear();
+
+        Boolean unionBool = checkBox.isSelected();
+
+        // add the geometries of the two polygon graphics to a list of geometries
+        List<Geometry> allPolygonGeometries = new ArrayList<>();
+        allPolygonGeometries.add(polygonGraphic1.getGeometry());
+        allPolygonGeometries.add(polygonGraphic2.getGeometry());
+
+        // retrieve the returned result from the convex hull operation.
+        // when unionBool = true there will be one returned convex hull geometry
+        // when = false, one convex hull geometry per input geometry.
+        List<Geometry> convexHullGeometries = GeometryEngine.convexHull(allPolygonGeometries, unionBool);
+
+        // loop through the returned geometries.
+        for (Geometry geometry : convexHullGeometries) {
+
+          // create a simple line symbol for the outline of the convex hull graphic(s)
+          SimpleLineSymbol convexHullLine = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 5);
+          // Create the simple fill symbol for the convex hull graphic(s)
+          SimpleFillSymbol convexHullFill = new SimpleFillSymbol(SimpleFillSymbol.Style.NULL, 0x00000000, convexHullLine);
+
+          // create the graphic for the convex hull(s)
+          Graphic convexHullGraphic = new Graphic(geometry, convexHullFill);
+
+          // add the convex hull(s) graphic to the convex hull graphics overlay
+          convexHullGraphicsOverlay.getGraphics().add(convexHullGraphic);
+        }
+
+        // enable reset button after convex hull button has been pressed.
+        resetButton.setDisable(false);
+        // Reset check box after convex hull button has been pressed.
+        checkBox.setSelected(false);
+      });
+
+      // disable clear button from starting application (when nothing to clear)
+      resetButton.setDisable(true);
+
+      resetButton.setOnAction(e -> {
+        // clear convex hull graphics from map view
+        convexHullGraphicsOverlay.getGraphics().clear();
+        // once cleared disable reset button
+        resetButton.setDisable(true);
+        // enable convex hull button after clearing
+        convexHullButton.setDisable(false);
+        // reset checkbox
+        checkBox.setSelected(false);
+      });
+
+      // create label
+      Label informationLabel = new Label(
```

Comment:
.NET samples shows instructions in their samples. Java chose not to do this because it clutters the UI and we already have the readme and Info tab in the sample viewer. So you can remove this label.